src/main/resources/hudson/plugins/git/extensions/impl/RelativeTargetDirectory/help-basename_zh_CN.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  对这里设置的子目录取其basename。<p>
```

Comment:
All the Chinese localization files should be put into [localization-zh-cn-plugin](https://github.com/jenkinsci/localization-zh-cn-plugin) according to [JEP-216](https://github.com/jenkinsci/jep/blob/master/jep/216/README.adoc).